script-base.js

```diff
@@ -73,15 +74,53 @@ var ScriptBase = yeoman.generators.NamedBase.extend({
   },
 
   _canGenerateTests: function () {
-    return this.config.get('testFramework') === 'mocha' && !this.config.get('includeRequireJS');
+    return this.config.get('testFramework') === 'mocha';
   },
 
-  _generateTest: function (type) {
+  _getMochaUi: function() {
+    var ui = this.config.get('ui');
+
+    if (!ui) {
+      try {
+        ui = require(path.join(process.cwd(), '.yo-rc.json'))['generator-mocha'].ui;
```

Comment:
@mteodori here's the conversation with Simon, taken from the previously referred gitter channel:

```
ruyadorno 15:24
@SBoudrias is there a way for parent level generators to access configstored data of nested/composed generators?

SBoudrias 15:53
@ruyadorno nope

ruyadorno 16:44
is there any better way to achieve something similar to this: https://github.com/yeoman/generator-backbone/pull/393/files#diff-fefbc4eae6b3c05767a1de4a95c9a67cR85 ?

SBoudrias 17:47
@ruyadorno I'd get then UI inside generator-backbone and pass it down to generator-mocha
@ruyadorno also, do you really need to ask for the UI? No one uses the tdd syntax

ruyadorno 17:48
ehehhehe good point
```

It seems to me that it's a much easier solution to just hardcode the `ui` option to always be `bdd`, what do you think?